<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_nova_venda</class>
 <widget class="QDialog" name="tela_nova_venda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource_file.qrc">
    <normaloff>:/logo cin.png</normaloff>:/logo cin.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(230, 230, 230);
color:black;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>Vendedor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_funcionario">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>12</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Georgia</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_funcionario">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Produto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_codigo">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_produto">
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color:rgb(0,0,0)</string>
         </property>
         <property name="text">
          <string>Quantidade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_qtd">
         <property name="styleSheet">
          <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color:rgb(0,0,0)</string>
         </property>
         <property name="text">
          <string>Valor Unit√°rio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_valor_un">
         <property name="styleSheet">
          <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_total">
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;</string>
     </property>
     <property name="text">
      <string>Total: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>480</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: white;
    color: black;
    gridline-color: #e1e1e1;
	border: 2px solid black
}

QTableWidget::item:selected {
    background-color: #ffcccc; 
    color: black;
}

QTableWidget::horizontalHeader {
    background-color: #f0f0f0;
    border: none;
}

QTableWidget::verticalHeader {
    background-color: #f0f0f0;
    border: none;
}

QTableWidget::item {
    padding: 5px;
}
</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_delete">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;
}

QPushButton:hover {
    background-color: rgb(255, 131, 131);
}

QPushButton:pressed {
    background-color: rgb(255, 131, 131);
}
</string>
       </property>
       <property name="text">
        <string>Deletar </string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="btn_limpar">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;
}

QPushButton:hover {
    background-color: rgb(255, 131, 131);
}

QPushButton:pressed {
    background-color: rgb(255, 131, 131);
}
</string>
       </property>
       <property name="text">
        <string>Limpar </string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_confirmar">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;
}

QPushButton:hover {
    background-color: rgb(255, 131, 131);
}

QPushButton:pressed {
    background-color: rgb(255, 131, 131);
}
</string>
       </property>
       <property name="text">
        <string>Confirmar </string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource_file.qrc"/>
 </resources>
 <connections/>
</ui>
